analysis/model/src/main/kotlin/de/maibornwolff/codecharta/serialization/OutputFileHandler.kt

```diff
@@ -19,16 +19,20 @@ object OutputFileHandler {
     }
 
     fun checkAndFixFileExtension(outputName: String): String {
-        if (outputName.endsWith("cc.json")) {
+        if (outputName.endsWith("cc.json") || outputName.endsWith(".cc.json.gz")) {
```

Comment:
I would just use a regular expression for cases like these. That way it's a single check with  optional parts. I would also handle the `.cc` optional (and our logic here is somewhat flawed as it would match files like `abcc.json`. I would accept it as it's a json file but it's just inconsistent right now.

Reply:
I think I remember @ce-bo saying that we should only consider .cc.json as valid, but thanks a lot for the hint with the regEx!
@phanlezz also noticed that we append .gz again, even if the user already specified it (so test.cc.json.gz becomes test.cc.json.gz.gz)
Should we/I open a new ticket for this or continue this here?